processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -1700,6 +1712,78 @@ public BufferComparator getBufferComparator()
         return bufferComparator;
       }
     }
+
+    // This class is only used when SQL compatible null handling is enabled.
+    private class NullableRowBasedKeySerdeHelper implements RowBasedKeySerdeHelper
```

Comment:
Please add a comment about the memory layout.

Reply:
done.